wdl-ast/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+#### Fixed
+
+* If a WDL >=v1.2 workflow has a non-boolean value for the `allow_nested_inputs` hint, the metadata will also be
+    searched for `allow_nested_inputs: <bool>` instead of prematurely dis-allowing nested inputs
```

Comment:
```suggestion
* If a WDL >=v1.2 workflow has a non-boolean value for the "allow nested inputs" hint, the metadata will also be
    searched for `allowNestedInputs: <bool>` instead of prematurely dis-allowing nested inputs
```